go/caddy/pfpki/cloud/intune.go

```
@@ -107,35 +109,21 @@ func (cl *Intune) NewCloud(ctx context.Context, name string) error {
 		}
 	}
 
-	oauthConfig, err := adal.NewOAuthConfig(activeDirectoryEndpoint, cl.TenantID)
-
-	// Intune token
-
-	spt, err := adal.NewServicePrincipalToken(*oauthConfig, cl.ClientID, cl.ClientSecret, graphResourceUrl)
-
-	err = spt.Refresh()
-
-	var token adal.Token
-
+	cred, err := azidentity.NewClientSecretCredential(cl.TenantID, cl.ClientID, cl.ClientSecret, nil)
```

Is it safe to ignore this error?